cli/lib/kontena/cli/vault/import_command.rb

```diff
@@ -0,0 +1,58 @@
+module Kontena::Cli::Vault
+  class ImportCommand < Kontena::Command
+    include Kontena::Cli::Common
+    include Kontena::Cli::GridOptions
+
+    option ['-u', '--upsert'], :flag, 'Create secret unless already exists', default: false
+    parameter "FILENAME", "Secret yaml file"
+
+    def execute
+      require_api_url
+      require_current_grid
+      token = require_token
+
+      # Not sure of a better way to do this for now. Skips the first two ARGF
+      # reads in order to get to the third without raising an error.
+      2.times do
+        begin
+          ARGF.read
+        rescue
+        end
+      end
+
+      raw = ARGF.read
+      exit_with_error('No data recieved from yaml file') if raw.to_s == ''
+      begin
+        secrets = YAML.load(raw)
+      rescue
+        exit_with_error('STDIN did not contain valid yaml')
+      end
+
+      current_secret_keys =
```

Comment:
You could just run the vault update command like this:
```ruby
result = Kontena.run(['vault', 'update', '--upsert', name, secret])
count += 1 if result == 0
```

instead of reimplementing the vault reading + writing here